flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/enumerator/initializer/OffsetsInitializerTest.java

```diff
@@ -52,9 +53,11 @@ public static void setup() throws Throwable {
                         KafkaSourceTestEnv.getAdminClient(), KafkaSourceTestEnv.GROUP_ID);
     }
 
-    @AfterClass
+    @AfterAll
     public static void tearDown() throws Exception {
```

Comment:
```suggestion
    static void tearDown() throws Exception {
```
could be package private